README.md

```diff
@@ -74,16 +74,22 @@ Need:
 
 - Cake 2: <http://bakery.cakephp.org/articles/lorenzo/2011/10/16/cakephp_2_0_released>
 
+- Typo3 Flow 1: <http://en.wikipedia.org/wiki/TYPO3_Flow>
```

Comment:
Most of the links are to the blog posts of the release or can be taken from archive.org